foot-in-the-door/pages/index.js

```diff
@@ -114,6 +109,11 @@ function Home(props) {
               </a>
             </figcaption>
           </figure>
+          <Link href="/pages/video">
```

Comment:
`/pages` never goes in the href of a Link. It's the directory the files live in, but never shows to the public. Everything inside of `pages/` becomes `<site URL>/<page name>`, so to make this link work it needs to be `<Link href="/video">`